.github/workflows/ci.yml

```diff
@@ -1,4 +1,4 @@
-name: Lint
+name: CI
```

Comment:
Might be helpful to give it a useful name

Reply:
Good point, I will find a better name